index.html

```diff
@@ -36,6 +36,196 @@ <h1>
             <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
         </ul>
     </section>
+
+    <section id="works">
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <span class="heading">Tonic</span>
+                </div>
+                <ul class="projectTools">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button class="enabled">See project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio1.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <span class="heading">Multi-Post
+                        Stories</span>
+                </div>
+                <ul class="projectTools">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
```

Comment:
- [ ] As suggested by previous reviewer and demanded by project requirements, kindly use an unordered list to list out technologies of each work card. the technologies list should look like
```html
<ul>
      <li>HTML</li>
      <li>CSS</li>
      <li>Javascript</li>
</ul>
```